packages/forklift-console-plugin/src/modules/Providers/views/create/components/VSphereProviderCreateForm.tsx

```diff
@@ -19,19 +19,22 @@ export const VSphereProviderCreateForm: React.FC<VSphereProviderCreateFormProps>
   const url = provider?.spec?.url || '';
   const vddkInitImage = provider?.spec?.settings?.['vddkInitImage'] || '';
 
-  const helperTextInvalid = {
+  const helperTextMsgs = {
     error: t(
-      'Error: Please provide a valid URL with a schema, domain, and path. For example: https://vcenter.com/sdk.',
+      'Error: The provided URL is in an invalid format. Please provide a URL that includes a scheme, a domain name and path. For example: https://vCenter-host-example.com/sdk',
     ),
     warning: t(
-      'Warning: The provided URL does not end with "sdk". Ensure it includes the correct path, like: https://vcenter.com/sdk.',
+      'Warning: The provided URL does not end with the API endpoint path: "sdk". Ensure it includes the correct path, For example: https://vCenter-host-example.com/sdk',
+    ),
+    success: t(
+      'URL of the vCenter API endpoint. Ensure it includes the "/sdk" path, for example: https://vCenter-host-example.com/sdk',
```

Comment:
```suggestion
      'URL of the vCenter SDK API endpoint. Ensure it includes the "/sdk" path, for example: https://vCenter-host-example.com/sdk',
```

AFAIU the SDK is is important information when describing the API endpoint, @ahadas @RichardHoch  WDYT ?

Reply:
Either "Ensure it includes the "/sdk" path. For example: ..."   or "Ensure it includes the "/sdk" path, for example, ..."